language/documentation/book/src/integers.md

```diff
@@ -21,23 +23,29 @@ If a literal is too large for its specified (or inferred) size range, an error i
 let explicit_u8 = 1u8;
 let explicit_u64 = 2u64;
 let explicit_u128 = 3u128;
+let explicit_u8_underscored = 1_2_3u8;
+let explicit_u64_underscored = 24_5_u64;
+let explicit_u128_underscored = 3000_0__000_1u128;
```

Comment:
I think the examples with multiple `__`'s and trailing `_`'s are funky--would leave them out. I would suggest just one or two "realistic" examples that show the value of the feature.